.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Let's also add `.DS_Store` to this list.

Reply:
One you've done that, you might also have to manually remove and commit the removal of the `.DS_Store` file since it was previously added.

Reply:
I see the `.gitignore` addition, but you'll still have to delete the file since it was previously added and commit that deletion.